test/provider_test.c

```diff
@@ -32,7 +32,7 @@ static int test_provider(const char *name)
     const char *greeting = NULL;
     char expected_greeting[256];
 
-    snprintf(expected_greeting, sizeof(expected_greeting),
+    BIO_snprintf(expected_greeting, sizeof(expected_greeting),
              "Hello OpenSSL %.20s, greetings from %s!",
              OPENSSL_VERSION_STR, name);
```

Comment:
Style nit...

Reply:
Oops: fixed in https://github.com/openssl/openssl/pull/8580